packages/xstate-test/src/TestModel.ts

```diff
@@ -283,28 +283,21 @@ export class TestModel<TState, TEvent extends EventObject> {
         testPathResult.steps.push(testStepResult);
 
         try {
-          await this.testState(params, step.state, options);
+          await this.testTransition(params, step);
```

Comment:
would this somehow allow for testing `xstate.init`? do we imagine it being useful?

Reply:
Yes it would, which can be very useful

Reply:
could u show an example of that?